resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/submit/v2/Client.scala

```diff
@@ -227,8 +235,10 @@ private[spark] class Client(
       resolvedLocalClassPath.foreach { classPathEntry =>
         require(Option(URI.create(classPathEntry).getScheme).isEmpty)
       }
-      val resolvedDriverJavaOpts = resolvedSparkConf.getAll.map { case (confKey, confValue) =>
-          s"-D$confKey=$confValue"
+      sparkConfExecutorsFetchRemoteDeps.set(
+        EXECUTOR_RESOLVED_MOUNTED_CLASSPATH, resolvedLocalClassPath)
```

Comment:
I'm not entirely sure if this is the best way to pass this information along. The submission client here has already resolved what the local classpath should be and assumes that the same localized classpath would be valid on the executors. This is similar to what the YARN submission client does, but it would be nicer to have the driver itself be responsible for determining the classpath of its executors. However, perhaps having such logic on the driver is repetitive if the submission client had already determined what the resolved classpath would be.